packages/components/pie-modal/src/index.ts

```diff
@@ -8,10 +8,10 @@ import {
 } from '@justeattakeaway/pie-webc-core';
 import type { DependentMap } from '@justeattakeaway/pie-webc-core';
 import { disableBodyScroll, enableBodyScroll } from 'body-scroll-lock';
-import '@justeattakeaway/pie-icons-webc/icons/IconClose';
-import '@justeattakeaway/pie-icons-webc/icons/IconChevronLeft';
-import '@justeattakeaway/pie-icons-webc/icons/IconChevronRight';
 import { Variant } from '@justeattakeaway/pie-button/src/defs.ts';
+import '@justeattakeaway/pie-icons-webc/dist/icons/IconClose.js';
+import '@justeattakeaway/pie-icons-webc/dist/icons/IconChevronLeft.js';
+import '@justeattakeaway/pie-icons-webc/dist/icons/IconChevronRight.js';
```

Comment:
is there a reason why we are importing the dist folder and js files here?

Reply:
Yes, because it doesn't work with the TS files 😅
Ash has agreed that we can look more into this in future

Reply:
could you please create a ticket for it so we don't lose track? 🔆

Reply:
I don't mind doing this, but what do you think the ticket should do? IMO if we do make one it should be more of an investigation than a task.
Ultimately they're the same code, just one is TS and one is JS. Also, [Spectrum seem to import components as JS](https://github.com/adobe/spectrum-web-components/blob/main/packages/dialog/src/Dialog.ts#L28-L31) so I don't think it's a bad thing to be doing, especially as the browser will be wanting JS anyway.

Reply:
I'll make a ticket for this today, as I wanted to have a play around with the paths once this is merged. I'll link it here once I've created it

Reply:
sorry if it wasn't clear, my point isn't about using js files only, more about the import itself as Ash mentioned.